src/lib.rs

```diff
@@ -32,7 +32,11 @@ use std::time::Duration;
 use std::{env, str::Utf8Error};
 
 use hyper::{body::Buf, client::HttpConnector, Body, Method};
+#[cfg(any(feature = "rustls-native", feature = "rustls-webpki"))]
```

Comment:
nit: fully qualifying the HttpsConnector below would be a bit clearer + get rid of these